meson.build

```
@@ -0,0 +1,95 @@
+# SPDX-FileCopyrightText: 2020 Josh Morman 
+#
+
+project('gnuradio', 'cpp', 
+  version : '4.0.0.0-preview0',
+  license : 'GPLv3',
+  default_options : ['cpp_std=c++17'])
+
+cc = meson.get_compiler('cpp')
+rt_dep = cc.find_library('rt', required : false)
+libdl_dep = cc.find_library('dl')
+
+c_available = add_languages('c', required : true)
+cuda_available = add_languages('cuda', required : false)
+USE_CUDA = cuda_available and get_option('enable_cuda')
+IMPLEMENT_CUDA = USE_CUDA
+IMPLEMENT_CPU = true
+
+SCRIPTS_DIR=join_paths(meson.project_source_root(),'utils','blockbuilder','scripts')
+
+volk_dep = dependency('volk', version : '>=2.2')
+yaml_dep = dependency('yaml-cpp', version : '>=0.6')
+fmt_dep = dependency('fmt', method: 'cmake', modules: ['fmt::fmt'])
+spdlog_dep = dependency('spdlog', method: 'cmake', modules : ['spdlog::spdlog'])
+python3_dep = dependency('python3', required : get_option('enable_python'))
+python3_embed_dep = dependency('python3-embed', required : get_option('enable_python'))
+
+# Import python3 meson module which is used to find the Python dependencies.
+py3_inst = import('python').find_installation('python3')
+py3_mod = import('python3')
+py3 = py3_mod.find_python()
+py3_version = py3_mod.language_version()
+if py3_version.version_compare('< 3.6')
+  error('Invalid python version!?')
+endif
+
+pybind11_dep = dependency('pybind11', required : get_option('enable_python'))
+threads_dep = dependency('threads')
+gtest_dep = dependency('gtest', main : true, version : '>=1.10', required : get_option('enable_testing'))
+cuda_dep = dependency('cuda', version : '>=10.1', required : USE_CUDA, modules : ['cufft'])
+
+libpmtf = subproject('pmt')
+pmtf_dep = libpmtf.get_variable('pmtf_dep')
+
+libCLI11 = subproject('CLI11')
+CLI11_dep = libCLI11.get_variable('CLI11_dep')
```

... and you can change this to
```suggestion
CLI11_dep = dependency('CLI11')
```

---

I'm going to postpone the `meson wrap` changes to a separate PR - this one gets a bit complicated by the fact that PMT uses CLI11 as well - want to fix that one first

---

FWIW, the first project to configure the subproject "wins".

It's an easy fix anyway, but I found a couple more: https://github.com/gnuradio/pmt/pull/52